doc/stats.rst

```diff
@@ -0,0 +1,106 @@
+
+Overview
+--------
+
+`pairtools stats` produces a human-readable nested dictionary of statistics stored in
+a YAML file or a tab-separated text table (specified through the parameters).
+
+- **Summary statistics** include:
+    - number of pairs (total, unmapped, single-side mapped, etc.),
+    - total number of different pair types (UU, NN, NU, and others, see ` Pair types in pairtools docs <https://pairtools.readthedocs.io/en/latest/formats.html#pair-types>`_),
+    - fraction of duplicates
+    - fraction of cis interactions out of total
+    - fraction of contacts between certain chromosomes
+
+Summary statistics can inform you about the quality of the data.
+For example, more trans interactions can be a sign of Hi-C of protocol problems and lower signal-to-noise ratio.
+Substantial mapping to mitochondrial chromosome (chrM) might be a sign of random ligation.
+
+- **P(s), or scaling.**  The dependence of contact frequency on the genomic
+distance referred to as the P(s) curve or scaling, which is a rich source of information of information and quality of 3C+ experiments.
+The shape of P(s) is often used to characterize mechanisms of genome folding and reveal issues with QC.
+
+Interactive visualization of stats with MultiQC
+---------
+
+Install `multiqc`:
+
+.. code-block:: bash
+
+    pip install --upgrade --force-reinstall git+https://github.com/open2c/MultiQC.git
+
+Run MultiQC in a folder with one or multiple .stats files:
+
+.. code-block:: bash
+
+    multiqc .
+
+
+This will produce nice .html file with interactive graphical summaries of the stats.
+
+
+Estimating library complexity
+----------------------------
+
+Pairtools assumes that each sequencing read is randomly chosen with
+replacement from a finite pool of fragments in DNA library [1]_ [2]_.
+With each new sequenced molecule, the expected number of observed unique molecules
+increases according to a simple equation:
+
+$$ U(N+1) = U(N) + (1 - {U(N) \\over C}), $$
+
+where $N$ is the number of sequenced molecules, $U(N)$ is the expected number
+of observed unique molecules after sequencing $N$ molecules, and C is the library complexity.
+This differential equation yields [1, 2]:
+
+$$ {U(N) \\over C} = 1 - exp( - {N \\over C}), $$
+
+which can be solved as
+
+$$ C = \Re(lambert W( - { \exp( - {1 \\over u} ) \\over u} ) ) + {1 \\over u} $$
+
+Library complexity can guide in the choice of sequencing depth of the library
+and provide an estimate of library quality.
+
+
+Illumina sequencing duplicates
+-----------------
+
+Importantly, you can estimate the complexity of Hi-C libraries using only small QC
+samples to decide if their quality permits deeper sequencing [3]_.
+These estimates, however, can be significantly biased by the presence of “optical” or
+“clustering” duplicates. Such duplicates occur when a DNA sequencer erroneously splits
+a signal from a single sequenced molecule into two; alternatively, a molecule located between
```

Comment:
Clustering duplicates aren't really explained here, there the cluster generation procedure can generate clusters from the same fragment in neighbouring nanowells on patterned flowcells. The difference with optical duplicates is that there there are no nanowells at all, and the issue occurs when a cluster has an irregular shape and the image analysis software interprets them as two separate clusters.